core/config/docs/chains-evm.toml

```
@@ -469,3 +469,5 @@ GasLimit = 5400000 # Default
 FromAddress = '0x2a3e23c6f242F5345320814aC8a1b4E58707D292' # Example
 # ForwarderAddress is the keystone forwarder contract address on chain.
 ForwarderAddress = '0x2a3e23c6f242F5345320814aC8a1b4E58707D292' # Example
+# DefaultGasLimit is the default gas limit for workflow transactions.
+DefaultGasLimit = 400_000 # Default
```

Keeping modifiers as suffixes makes things a little easier to read and keeps things grouped together naturally when sorted:
```suggestion
GasLimitDefault = 400_000 # Default
```

---

e.g.: https://github.com/smartcontractkit/chainlink/pull/14214/files#diff-525ad0c1ca7121f3a6072b304276f56ea2bd8f05e3d7dc243f245ea500793375R169-R195